x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/translations.ts

```diff
@@ -18,7 +18,7 @@ export const IMPORT_RULE = i18n.translate('xpack.siem.detectionEngine.rules.impo
 });
 
 export const ADD_NEW_RULE = i18n.translate('xpack.siem.detectionEngine.rules.addNewRuleTitle', {
-  defaultMessage: 'Add new rule',
+  defaultMessage: 'Create new rule',
```

Comment:
Missed `BACK_TO_DETECTION_ENGINE` above (line 12) which still says `'Back to detection engine'` instead of `'Back to detections'`:

https://github.com/elastic/kibana/blob/a7c5dbfc7fd84557e504eadf227755e8818f9115/x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/translations.ts#L12